### ISSUE TITLE
Change the dependency on jQuery not to allow jQuery 3?

### ISSUE THREAD

**Issue by linsolas** [eternicode/bootstrap-datepicker#1921]
Hello,

Bootstrap 3 is [not compatible with jQuery 3](https://github.com/twbs/bootstrap/issues/16834), and may not be corrected to support that version.
Unfortunately, bootstrap-datepicker allows [any version of jQuery superior to 1.7.1](https://github.com/eternicode/bootstrap-datepicker/blob/master/package.json#L27-L29):

```
  "dependencies": {
    "jquery" : ">=1.7.1"
  }
```

Shouldn't bootstrap-datepicker restrict the jQuery version, something like that:

```
  "dependencies": {
    "jquery" : ">=1.7.1 <3.0.0"
  }
```

Thanks

**Comment by acrobat** [comment]
Yes that's indeed a too open constraint! Can you open a PR for that? If you open a PR please also update the constraint in `bower.json` and `composer.json`

**Comment by linsolas** [comment]
Thanks. I wanted to have a collaborator opinion before. I've made the #1922 for that.

**Comment by acrobat** [comment]
:+1:. Fixed in #1922
